### PR TITLE
delete particular table.

### PR DESCRIPTION
**Pull request by k-morozov**
Previously conversation https://github.com/yandex/ch-tools/pull/182#discussion_r1607804879

### CODE REVIEW

File: ch_tools/chadmin/cli/table_group.py [hunk 1]
```diff
@@ -297,7 +323,26 @@ def delete_command(
...
+            table_name=table_name,
+            cluster=cluster,
+            sync_mode=sync_mode,
+            echo=True,
+            dry_run=dry_run,
+        )
+        return
+
+    tables = list_tables(ctx, **kwargs)
+    if 0 == len(tables):
```
Comment (MikhailBurdukov) [inline ch_tools/chadmin/cli/table_group.py:340]:
```suggestion
    if len(tables):
```

File: ch_tools/chadmin/cli/table_group.py [hunk 2]
```diff
@@ -297,7 +323,26 @@ def delete_command(
         )
         return
 
-    for t in list_tables(ctx, **kwargs):
+    if database_name is not None and table_name is not None:
```
Comment (MikhailBurdukov) [inline ch_tools/chadmin/cli/table_group.py:326]:
```suggestion
    if database_name and table_name:
```

File: tests/features/table_delete.feature [hunk 3]
```diff
@@ -0,0 +1,261 @@
+Feature: chadmin delete detached table commands
```
Comment (MikhailBurdukov) [inline tests/features/table_delete.feature:1]: Also lets add a test to cover cli options usage, suggest to use `Scenario Outline` for that.

File: ch_tools/chadmin/cli/table_group.py (hunk 1 above)
Comment (k-morozov) [reply ch_tools/chadmin/cli/table_group.py:340]: Why? It will be another result.

Comment (MikhailBurdukov) [reply ch_tools/chadmin/cli/table_group.py:340]: Sorry 
```suggestion
    if not len(tables):
```

File: tests/features/table_delete.feature (hunk 3 above)
Comment (k-morozov) [reply tests/features/table_delete.feature:1]: I think your suggestion is mostly relate with test coverage of existing functionality (table delete by pattern) than current adding feature of deleting particular table. So we have a lot of variants of using args/options, checking constraints when we use command table delete. I suppose it's better to do it in a separate task, if you dont mind.

**Review by MikhailBurdukov — APPROVED** [review]

**Pull request merged**